bundles/org.openhab.binding.tr064/src/main/resources/channels.xml

```diff
@@ -244,6 +196,24 @@
 			serviceId="urn:WANCIfConfig-com:serviceId:WANCommonInterfaceConfig1"/>
 		<getAction name="GetTotalBytesSent" argument="NewTotalBytesSent"/>
 	</channel>
+	<channel name="wanCurrentDownstreamBitrate" label="Current Downstream Rate">
```

Comment:
Thing upgrade instructions seems missing for these new channels?

Reply:
The `channels.xml` only defines the channels. All channels are added/removed dynamically based on what the device reports as supported. This check is done during initialization of the handler (every time, not only the first time).